### PR TITLE
Add detainee details on profile page

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

File: app/presenters/detainee_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  def short_gender
+    if gender == 'male'
+      'M'
+    elsif gender == 'female'
+      'F'
+    end
+  end
+
+  def age
+    # TODO: Add validation on date_of_birth and remove presence check
```
Comment (whoojemaflip) [inline app/presenters/detainee_presenter.rb]: what's with the TODO?

Comment (cesidio) [inline app/presenters/detainee_presenter.rb]: All the fields in the detainee must be provided, whether if they come from NOMIS or by the prison staff. This is just a remainder that we have to implement that logic sooner or later. They will ask for it.

Comment (whoojemaflip) [inline app/presenters/detainee_presenter.rb]: Surely that will live in the Form object, not the presenter?

**Comment by whoojemaflip** [comment]
:+1:

**Pull request merged**